### PR TITLE
 Added Push Notification feature using firebase

### PR DESCRIPTION
**Pull request by anjalisoni3655**
- The user can receive a push notification in their device, clicking on the notification will redirect them to the home insurance app. 
- It will also ensure that the user can receive the notification, even if the app is running in the background or the app is completely closed. 
- The company or the admin can customize the message to be send to the user themselves using the firebase cloud messaging service. They can also schedule the messages making it a recurring notification. Here is a snapshot of a dummy notification.
PN

### CODE REVIEW

**Comment by anjalisoni3655** [comment]
@googlebot I consent.

**Comment by dreamplay** [comment]
Let's not add this feature for now. keep it on hold

**Comment by anjalisoni3655** [comment]
> Let's not add this feature for now. keep it on hold

Can we add that now, since it's an insurance app, and the insurance company will tend to send those push notifications.

**Comment by anjalisoni3655** [comment]
@googlebot I consent.

**Comment by khushi2305** [comment]
@googlebot I consent.

File: home_insurance_app/pubspec.lock [hunk 1]
```diff
@@ -0,0 +1,694 @@
+# Generated by pub
```
Comment (dreamplay) [inline home_insurance_app/pubspec.lock:1]: this file isnt needed

**Pull request merged**